testing/injections/crypto_shim.js

```diff
@@ -1,37 +1,41 @@
 // ivm doens't have a crypto implementation. since we browserify modules already, this shim implements the browser crypto interface.
 // inspired by https://github.com/kumavis/polyfill-crypto.getrandomvalues/blob/master/index.js
 
-var MersenneTwister = require('mersenne-twister')
+var MersenneTwister = require('mersenne-twister');
+var semver = require('semver');
```

Comment:
Looks like the `semver` import here may be unused?

Reply:
Oh, yeah @alan-codaio can this be removed? It's bloating back bundles.

Reply:
yeah — I'll take it out

Reply:
https://github.com/coda/packs-sdk/pull/2469